roles/aap_setup_prepare/templates/inventory.j2

```diff
@@ -2,15 +2,12 @@
 
 {% for group_key in aap_setup_prep_inv_nodes %}
 
-{%if group_key in groups %}
```

Comment:
why this change, believe this would break the entire template, can you share how its breaking versus the examples.